internal/shell/runner.tmpl.sh

```diff
@@ -0,0 +1,312 @@
+#!/bin/sh
```

Comment:
installer and runner templates contain too many duplicate section. 

It's not maintainable. 
Please avoid coping and pasting. 

Option 1
Use single template file and use if-else of Go text template to change content. 

Option 2 
Utilize go text template feature. e.g. use sub template.